numpy/lib/function_base.py

```diff
@@ -56,24 +56,24 @@ def iterable(y):
 
     Returns
     -------
-    b : {0, 1}
-      Return 1 if the object has an iterator method or is a sequence,
-      and 0 otherwise.
+    b : {False, True}
+      Return ``True`` if the object has an iterator method or is a
+      sequence, and ``False`` otherwise.
 
 
     Examples
     --------
     >>> np.iterable([1, 2, 3])
-    1
+    False
     >>> np.iterable(2)
-    0
+    True
 
     """
     try:
         iter(y)
     except:
```

Comment:
Not your fault, but while you're here, might as well clean this up too... `except:` is never correct (e.g. it swallows `KeyboardInterrupt` exceptions). `except Exception` would be better, and I guess `except TypeError` would be better yet...